### PR TITLE
Improve 0.6.0 release notes

### PR DESCRIPTION
**Pull request by garethbowen**
medic/medic-android#137

### CODE REVIEW

**Review by kennsippell — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -16,7 +16,11 @@ In order to comply with Play Store policy it was necessary to switch from using
 1. When loading the application after upgrade you will need an internet connection in order to cache the application code.
 2. The migration can take a few minutes to complete so after upgrade the user may be shown a login screen. If this happens, restart the application periodically until the user is logged in and the app loads as per usual.
 
-Users on Android 9 and below will continue to use crosswalk and be unaffected by this change.
+We are planning improvements to [make the migration more user friendly](https://github.com/medic/medic-android/issues/134) in a future release.
```
[the remark this answers is not in the document]
Comment (garethbowen) [reply README.md:19]: The section you wrote was similar to, but better than, my previous introduction, so I copied it up there.

**Comment by garethbowen** [comment]
Thanks @kennsippell . Would you mind reviewing the latest version?

File: README.md [hunk 2]
```diff
@@ -11,12 +11,16 @@ The medic-android application is a thin wrapper to load the [CHT Core Framework]
...
+This release is largely intended to unblock the publishing of new apps onto the Google Play Store in a manner compatible with Android 10+. This release includes a difficult upgrade experience which may cause operational challenges in the field for apps with users on Android 10+. In particular:
 
 1. When loading the application after upgrade you will need an internet connection in order to cache the application code.
 2. The migration can take a few minutes to complete so after upgrade the user may be shown a login screen. If this happens, restart the application periodically until the user is logged in and the app loads as per usual.
 
-Users on Android 9 and below will continue to use crosswalk and be unaffected by this change.
+We are planning improvements to [make the migration more user friendly](https://github.com/medic/medic-android/issues/134) in a future release.
+
+Users on Android 9 and below will do not need to be migrated and will be unaffected by this change. Because of this it is recommended that projects upgrade to v0.6.0 version of their app before issuing Android 10+ devices, if possible.
```
Comment (dianabarsan) [inline README.md:21]: Typo here `will do not need`

**Review by dianabarsan — APPROVED** [review]
👍🏻

File: README.md (hunk 2 above)
Comment (garethbowen) [reply README.md]: Fixed.

**Pull request merged**